simpleio.py

```diff
@@ -190,17 +209,17 @@ class DigitalOut:
       :param drive_mode digitalio.DriveMode: drive mode for the output
     """
 
-    def __init__(self, pin, **kwargs):
+    def __init__(self, pin: Pin, **kwargs: Any) -> None:
         self.iopin = digitalio.DigitalInOut(pin)
         self.iopin.switch_to_output(**kwargs)
 
     @property
-    def value(self):
+    def value(self) -> Any:
```

Comment:
I think this could be `bool` instead of `Any`

Reply:
Corrected in the latest push together with the next change request.
The issue here is that mypy will complain. Technically `self.iopin.value` will return a boolean as it evaluates from the below function in the digitalio package. However, until that function is type-hinted to return bool, mypy thinks it is returning Any, and will hence complain as shown.
```python
@property
def value(self):
    """The Digital Pin Value"""
    return self._pin.value() == 1
```
```
simpleio.py:219: error: Returning Any from function declared to return "bool"  [no-any-return]
simpleio.py:241: error: Returning Any from function declared to return "bool"  [no-any-return]
```

Reply:
Ahh, that makes sense. Thank you. 

That is okay to have this one raising an error here for that reason. We'll get digitalio typed eventually as well to resolve this. I wonder where it would pull types for built-in modules from, the stubs package, or perhaps Blinka implementation.